modules/dashboard/php/NDB_Form_dashboard.class.inc

```diff
@@ -138,10 +138,10 @@ class NDB_Form_Dashboard extends NDB_Form
                     "SELECT COUNT(*) FROM conflicts_unresolved cu 
                      LEFT JOIN flag ON (cu.CommentId1=flag.CommentID) 
                      LEFT JOIN session s ON (flag.SessionID=s.ID)
-		             LEFT JOIN candidate c ON (c.CandID=s.CandID)
+		     LEFT JOIN candidate c ON (c.CandID=s.CandID)
                      LEFT JOIN psc ON (psc.CenterID=s.CenterID) 
                      WHERE psc.Name=:Site
-			            AND s.Active='Y' AND c.Active='Y'",
+			s.Active='Y' AND c.Active='Y'",
```

Comment:
Looks like this should be rebased(?) - e.g. it's missing the critical 16.0.2. bug fix #2043